src/test/java/org/minions/devfund/sergio/movie_databaseTest/MovieDatabaseTest.java

```diff
@@ -0,0 +1,115 @@
+package org.minions.devfund.sergio.movie_databaseTest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.minions.devfund.sergio.movie_database.Actor;
+import org.minions.devfund.sergio.movie_database.Movie;
+import org.minions.devfund.sergio.movie_database.MovieDatabase;
+
+import java.util.ArrayList;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class MovieDatabaseTest {
+    private MovieDatabase mdb_1;
+    private MovieDatabase mdb_2;
+    private MovieDatabase mdb_3;
+
+    private Movie movie_1;
+    private Movie movie_2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "movie_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)